### PR TITLE
ReFS deduplication and compression

### PR DESCRIPTION
**Pull request by ManikaDhiman**

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@ManikaDhiman - You've submitted this PR to the public repo. Please submit the content to azure-stack-docs-pr instead.

File: azure-stack/hci/manage/refs-deduplication-and-compression.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+Days                                      : EveryDay
+Suspended                                 : True
+```
+
+---
+
+### Disable ReFS deduplication and compression on a volume
+
+Disabling ReFS deduplication and compression on a volume stops any in-progress runs and cancels future scheduled jobs. In addition, related volume metadata isn't  retained, and file change tracking is stopped. Disabling this feature doesn't undo deduplication or compression, as all the operations occur at the metadata layer. Overtime, the data returns to its original state as the volume incurs reads and writes. You can perform decompression operations using `ReFSUtil`<!--Do we need the last sentence here?-->.
+
```
Comment (alkohli) [inline azure-stack/hci/manage/refs-deduplication-and-compression.md:254]: Over time

File: azure-stack/hci/manage/refs-deduplication-and-compression.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+<!--Mas to help w/ current state-->
+
+### Sending stopped monitoring Event Tracing for Windows (ETW) events after disabling ReFS deduplication and compression on a volume.
+
+Once ReFS deduplication and compression is disabled on a volume, the ETW channel for ReFS deduplication logs repeated stopped monitoring events. However, we don't anticipate significant usage impact because of this issue.
+
+### Job failed event not logged if volume is moved to another node during compression.
+
+If the CSV is moved to another node while compression is in progress, the job failed event isn't logged in the ReFS deduplication channel. However, we don't anticipate significant usage impact because of this issue.
+
```
Comment (alkohli) [inline azure-stack/hci/manage/refs-deduplication-and-compression.md:319]: Should we take this to top level known issues articles

**Review by alkohli — COMMENTED** [review]
Let us also discuss TOC placement. This should go in the preview node along with other 23H2 content.

**Pull request closed without merging**